src/test/java/seedu/address/logic/parser/AddCommandParserTest.java

```diff
@@ -69,6 +69,7 @@ public void parse_allFieldsPresent_success() {
                         + TAG_DESC_FRIEND,
                 new AddCommand(expectedPerson));
 
+        //@@author sarahnzx
```

Comment:
Marking of codes should not be inside a method:

> If a method was written by more than one person, the //@@author tag should indicate the person who wrote most of that method. 
**i.e., //@@author tag should not appear inside a method.**